src/qt/diagnosticsdialog.cpp

```diff
@@ -178,7 +178,7 @@ void DiagnosticsDialog::VerifyTCPPort() {
     connect(tcpSocket, SIGNAL(connected()), this, SLOT(TCPFinished()));
     connect(tcpSocket, SIGNAL(error(QAbstractSocket::SocketError)), this, SLOT(TCPFailed(QAbstractSocket::SocketError)));
 
-    tcpSocket->connectToHost("london.grcnode.co.uk", 32749);
+    tcpSocket->connectToHost("seeds.gridcoin.ifoggz-network.xyz", 32749);
```

Comment:
Maybe we should use this or something similar: http://portquiz.net/

Reply:
i'll check it out i had put my server for testing it.